cohere/client.py

```diff
@@ -144,6 +144,37 @@ def choose_best(self, model: str, query: str, options: List[str], mode:  str = '
         response = self.__request(json_body, cohere.CHOOSE_BEST_URL, model)
         return BestChoices(response['scores'], response['tokens'], response['token_log_likelihoods'], mode)
 
+    def classify(
+        self,
+        model: str,
+        inputs: List[str],
+        examples: List[Example],
+        taskDescription: str = "",
+        outputIndicator: str = ""
+    ) -> Classifications:
```

Comment:
docstring this user facing function. 

if not possible for this PR, please create a github issue for later PR